operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -76,6 +76,14 @@ spec:
 #    ---
 #    strategy: ""
 
+##########
+# Determines whether to show links of Grafana and Traces or not depending on user
+# access to Istio control plane pods. Kiali won't show Grafana and Tracing integration
+# in case the user doesn't have permissions to list pods of IstioNamespace.
+# When empty, its default will be `true`.
+#    ---
+#  authorize_external_services: false
```

Comment:
Also, any settings you add must be added to the roles/kiali-deploy/defaults/main.yml. So add this there:

```
...
auth:
  strategy: ""

authorize_external_services: true

deployment:
...
```